metomi/rose/config_processors/fileinstall.py

```diff
@@ -856,7 +857,7 @@ def parse(self, loc, conf_tree):
             # Scheme specified in the configuration.
             handler = self.get_handler(loc.scheme)
             if handler is None:
-                raise ValueError(loc.name)
+                raise ValueError(f"don't support scheme {loc.scheme}")
```

Comment:
[minor] maybe a slightly cryptic error message for users to digest:

```suggestion
                raise ValueError(f"Rose doesn't support scheme {loc.scheme}")
```